hw/ip/dcd/dv/dcd_sim.core

```diff
@@ -0,0 +1,30 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:dcd_sim:0.1"
+description: "DCD DV sim target"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:ip:dcd
+
+  files_dv:
+    depend:
+      - lowrisc:dv:dcd_test
+      - lowrisc:dv:dcd_sva
+    files:
+      - tb.sv
+    file_type: systemVerilogSource
+
+targets:
+  sim: &sim_target
+    toplevel: tb
+    filesets:
+      - files_rtl
+      - files_dv
+    default_tool: vcs
+
+  # TODO: add a lint check cfg in `hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson`
```

Comment:
Would you mind addressing this TODO right away?

Reply:
will do.